includes/User.php

```diff
@@ -4218,6 +4218,20 @@ public static function crypt( $password, $salt = false ) {
 	 * @return Boolean
 	 */
 	public static function comparePasswords( $hash, $password, $userId = false ) {
+		// Wikia change - begin
+		// @see PLATFORM-2502 comparing new passwords in PHP code.
+		// @todo mech remove after the new password hashing is implemented.
+		$caller = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS)[1];
+		Wikia\Logger\WikiaLogger::instance()->debug(
```

Comment:
I believe Helios handles most of the password checking requests and logs the calls without any sampling (https://github.com/Wikia/app/blob/b03df0a89ed672697e9c130d529bf1eb25f49cda/extensions/wikia/Helios/User.class.php#L161). Additionally I've checked those logs on the dashboard (https://kibana.wikia-inc.com/index.html#/dashboard/elasticsearch/HELIOS_LOGIN) and it generates round 0.1 qps